### PR TITLE
Ignore ingame services for the auto-stop of the Smart Module

### PR DESCRIPTION
**Pull request by 0utplay**
This pull request includes:

- [ ] breaking changes
- [x] no breaking changes

### Changes made to the repository:

Ignore ingame services for the auto-stop of the Smart Module

### Documentation of test results:

No tests have been done so far.

### Related issues/discussions:

https://github.com/CloudNetService/CloudNet-v3/projects/2#card-39204647

### CODE REVIEW

**Comment by byRoadrunner** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=5833196&bid=19097590)

File: cloudnet-modules/cloudnet-smart/src/main/java/de/dytanic/cloudnet/ext/smart/listener/CloudServiceListener.java [hunk 1]
```diff
@@ -1,11 +1,13 @@
 package de.dytanic.cloudnet.ext.smart.listener;
 
 import de.dytanic.cloudnet.CloudNet;
+import de.dytanic.cloudnet.driver.CloudNetDriver;
```
Comment (byRoadrunner) [inline cloudnet-modules/cloudnet-smart/src/main/java/de/dytanic/cloudnet/ext/smart/listener/CloudServiceListener.java:4]: Codacy found an issue: [Unused import - de.dytanic.cloudnet.driver.CloudNetDriver.](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=5833196)

Comment (byRoadrunner) [inline cloudnet-modules/cloudnet-smart/src/main/java/de/dytanic/cloudnet/ext/smart/listener/CloudServiceListener.java:4]: Codacy found an issue: [Avoid unused imports such as 'de.dytanic.cloudnet.driver.CloudNetDriver'](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=5833196)

**Comment by derrop** [comment]
Well, that's not the auto stop, it's the maxServiceCount and that should not ignore ingame services.

**Pull request closed without merging**